src/main/java/mate/dao/impl/CarDaoImpl.java

```diff
@@ -237,7 +238,7 @@ private Car parseCarFromResultSet(ResultSet resultSet) throws SQLException {
         String manufacturerCountry = resultSet.getNString("manufacturer_country");
         Manufacturer manufacturer = new Manufacturer(manufacturerName, manufacturerCountry);
         manufacturer.setId(manufacturerId);
-        Long carId = resultSet.getObject("id", Long.class);
+        long carId = resultSet.getLong("id");
```

Comment:
return the previous version, look through all your code and fix it
```suggestion
        Long carId = resultSet.getObject("id", Long.class);
```